backend/app/services/home.py

```
@@ -0,0 +1,42 @@
+from typing import Any
+
+import httpx
+
+from app.core.config import get_settings
+from app.services.base import BaseHomeService
+
+settings = get_settings()
+
+
+class HomeAssistantService(BaseHomeService):
+    def __init__(self):
```

Ruff (0.14.7)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)